frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -67,106 +201,122 @@ export function TrainAiModelTab({ onClose }: { onClose: () => void }) {
-          type="warning"
-          showIcon
-        />
-      </Row>
-      <Row gutter={8}>
-        <Col span={24}>
-          <FormItem
-            hasFeedback
-            name="modelName"
-            label="Model Name"
-            rules={[
-              {
-                required: true,
-                message: "Please name the model that should be trained.",
-              },
-            ]}
-          >
-            <Input autoFocus />
-          </FormItem>
-        </Col>
-      </Row>
-      <FormItem
-        hasFeedback
-        name="modelCategory"
-        label="Model Category"
-        rules={[
-          {
-            required: true,
-            message: "Please select a model category.",
-          },
-        ]}
-      >
-        <Select>
-          <Select.Option value={AiModelCategory.EM_NEURONS}>EM Neurons</Select.Option>
-          <Select.Option value={AiModelCategory.EM_NUCLEI}>EM Nuclei</Select.Option>
-        </Select>
-      </FormItem>
-      <FormItem
-        hasFeedback
-        name="imageDataLayer"
-        label="Image Data Layer"
-        hidden={colorLayers.length === 1}
-        rules={[
-          {
-            required: true,
-            message: "Please select a layer whose image data should be used for training.",
-          },
-        ]}
-      >
-        <LayerSelection layers={colorLayers} tracing={tracing} style={{ width: "100%" }} />
-      </FormItem>
-      <LayerSelectionFormItem
-        chooseSegmentationLayer
-        layers={segmentationLayers}
-        fixedLayerName={fixedSelectedLayer?.name}
-        tracing={tracing}
-        label="Groundtruth Layer"
-      />
-      <Row gutter={8}>
-        <Col span={24}>
-          <FormItem hasFeedback name="comment" label="Comment">
-            <Input />
-          </FormItem>
-        </Col>
-      </Row>
+    <Form
+      onFinish={(values) => onFinish(form, useCustomWorkflow, values)}
+      form={form}
+      initialValues={defaultValues}
+      layout="vertical"
+    >
+      <ExperimentalWarning />
+      <AiModelNameFormItem />
+      <AiModelCategoryFormItem />
+
+      {annotationsWithDatasets.map(({ annotation, dataset }, idx) => {
+        const segmentationLayerNames = _.uniq([
+          // Only consider the layers that are not volume layers (these don't have a tracing id).
+          // Add actual volume layers below.
+          ...getSegmentationLayers(dataset)
+            .filter((layer) => layer.tracingId == null)
+            .map((layer) => layer.name),
+          // Add volume layers here.
+          ...annotation.annotationLayers
+            .filter((layer) => layer.typ === "Volume")
+            .map((layer) => layer.name),
+        ]);
+        const segmentationLayers: Array<{ name: string }> = segmentationLayerNames.map(
+          (layerName) => ({
+            name: layerName,
+          }),
+        );
+
+        const fixedSelectedLayer = segmentationLayers.length === 1 ? segmentationLayers[0] : null;
```

Comment:
Can you rename this to indicate that this is the fixed segmentation layer. While you're at it, you could also unify it with the fixing of the color layer (line 275). Either fix both in the props or both here.